test/fixtures/generateBreaks/generateRenderer-UniqueValue.json

```diff
@@ -0,0 +1,28 @@
+{
+  "classificationDef": {
```

Comment:
it's good that we're supporting the geoservices version of a classification definition. we also want to support a simplified version within winnow. basically the pattern is to shorten the key names and smooth over any weird stuff from the geoservices api